.ado/publish.yml

```diff
@@ -498,22 +494,19 @@ extends:
             ls $(System.DefaultWorkingDirectory)/target/npm/qsharp/*
           displayName: Display NPM Artifacts in Publishing Dir
 
-        - task: EsrpRelease@7
+        - task: EsrpRelease@9
           condition: succeeded()
-          inputs:
-            ConnectedServiceName: 'ESRP Signing Connection'
-            KeyVaultName: 'kv-aqua-esrp-001'
-            AuthCertName: 'EsrpAuthCert'
-            SignCertName: 'EsrpSignCert'
-            ClientId: '832c049d-cd07-4c1c-bfa5-c07250d190cb'
-            Intent: 'PackageDistribution'
-            ContentType: 'NPM'
-            FolderLocation: '$(System.DefaultWorkingDirectory)/target/npm/qsharp'
-            WaitForReleaseCompletion: true
-            Owners: '$(OwnerPersonalAlias)@microsoft.com'  # NB: Group email here fails the task with non-actionable output.
-            Approvers: 'billti@microsoft.com'
-            # Auto-inserted Debugging defaults:
-            ServiceEndpointUrl: 'https://api.esrp.microsoft.com'
-            MainPublisher: 'ESRPRELPACMAN'                 # Default ESRP v7 publisher. Do not change.
```

Comment:
You removed a few comments, are they no longer relevant?

Reply:
Using tool spec as given.